watertap/unit_models/tests/test_reverse_osmosis_0D.py

```diff
@@ -66,7 +66,7 @@ def test_config():
     m.fs.properties = props.NaClParameterBlock()
     m.fs.unit = ReverseOsmosis0D(property_package=m.fs.properties)
 
-    assert len(m.fs.unit.config) == 13
+    assert len(m.fs.unit.config) == 14
```

Comment:
Add tests to demo successful model creation, scaling, initialization, and solve, following other test examples for 0D and 1D.

Reply:
Should I create a new class for this or should I just add tests to the TestReverseOsmosis class?

Reply:
Looks like you handled it!